ios/LocalNotifyPlugin.mm

```diff
@@ -30,7 +30,7 @@ - (void) cancelOldNotifications {
 		UILocalNotification *n = [notifications objectAtIndex:ii];
```

Comment:
better to change it if it is easy, we will not revisit for a long time otherwise